LoopKitUI/CGMManagerUI.swift

```diff
@@ -19,7 +19,18 @@ public struct CGMManagerDescriptor {
     }
 }
 
-public protocol CGMManagerUI: CGMManager, DeviceManagerUI, PreferredGlucoseUnitObserver {
+public protocol DeviceStatusIndicator {
```

Comment:
include the status badge in here as well.

Reply:
Consider remaining to `CGMStatusIndicator`, since it isn't generic enough for any device.

Reply:
I agree with @nhamming 's suggestion to name this more specifically.